lib/ui/views/installer/installer_viewmodel.dart

```diff
@@ -38,6 +41,7 @@ class InstallerViewModel extends BaseViewModel {
   bool hasErrors = false;
   bool isCanceled = false;
   bool cancel = false;
+  bool doNotPopupScreenshotWarning = false;
```

Comment:
Rename this to showScreenshotWarning and negate the boolean logic. The code has double negations which make it confusing to understand.